email/registration.html

```diff
@@ -0,0 +1,26 @@
+<div>
```

Comment:
See the [sample](https://github.com/PureBhaktiArchive/audioseva/blob/release-1.0/email/sqr-allotment.html) that is linked in the card.

- Is there any `div`?
- What is the purpose of this `div`?
- What is the root element there in the sample?

These questions are not to be answered, just adjust your code.